freqtrade/fiat_convert.py

```diff
@@ -0,0 +1,161 @@
+import logging
+import time
+from pymarketcap import Pymarketcap
+
+logger = logging.getLogger(__name__)
+
+
+class PairConvertion():
+    # Constants
+    CACHE_DURATION = 6 * 60 * 60  # 6 hours
+
+    # Private attributes
+    crypto_symbol = None
+    fiat_symbol = None
+    price = 0.0
+    expiration = 0
+
+    def __init__(self, crypto_symbol: str, fiat_symbol: str, price: float) -> None:
+        """
+        Create an object that will contains the price for a crypto-currency in fiat
+        :param crypto_symbol:
```

Comment:
param descriptions are empty, please remove the params doc or fill them out :-)

Reply:
Updated. I forgot to complete it. Thanks for this review